app/assets/javascripts/angular/controllers/dashboard_ctrl.js

```diff
@@ -83,6 +83,10 @@ angular.module("Prometheus.controllers")
   };
 
   $document.keydown(function(ev) {
+    if (ev.keyCode === 90 && ev.metaKey) { // Z keycode + cmd key
```

Comment:
Ah, the Gmail approach sounds good too. Of course the "z" should not do anything when you're typing it anywhere in a text field :)